o/operator_utils/operator_utils_ubi8.5.sh

```diff
@@ -0,0 +1,84 @@
+# ----------------------------------------------------------------------------
+#
+# Package        : operator-utils
+# Version        : v1.1.1
+# Source repo    : https://github.com/redhat-cop/operator-utils
+# Tested on      : RHEL 8.4
+# Language       : GO
+# Script License : Apache License, Version 2 or later
+# Maintainer     : Sapna Shukla <Sapna.Shukla@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+
+CWD=`pwd`
+
+PACKAGE_PATH=github.com/redhat-cop/
+PACKAGE_NAME=operator-utils
+PACKAGE_VERSION=v1.1.1
+PACKAGE_URL=https://github.com/redhat-cop/operator-utils
+
+# Install dependencies
+yum install -y make git wget gcc
+
+# Download and install go
+wget https://golang.org/dl/go1.14.13.linux-ppc64le.tar.gz
```

Comment:
is there are a specific need for this go version as opposed to the latest or the distro default one?

Reply:
Changed this to 17.5